packages/telemetry/src/span-exporters/cloud-span-exporter.ts

```diff
@@ -0,0 +1,119 @@
+import type { ReadableSpan } from '@opentelemetry/sdk-trace-base'
+
+import type {
+  OTLPExporterNodeConfigBase,
+  OTLPExporterError,
+} from '@opentelemetry/otlp-exporter-base'
+
+import { diag } from '@opentelemetry/api'
+
+import { OTLPTraceExporter as OTLPTraceExporterHttp } from '@opentelemetry/exporter-trace-otlp-http'
+
+import { sendWithHttp } from '@opentelemetry/otlp-exporter-base'
+
+export interface OTLPExporterNodeConfigBasePlusEncryption extends OTLPExporterNodeConfigBase {
+  encryption?: {
+    encryptRequest: (requestOptions: {
+      url: string
+      method: string
+      body: string
+    }) => (Promise<{jwe: string}>)
+  }
+}
+
+/**
+ * Collector Trace Exporter for Node
+ */
+export class OTLPTraceExporter extends OTLPTraceExporterHttp {
+  delayedItemsToExport: {
+    serviceRequest: string
+    onSuccess: () => void
+    onError: (error: OTLPExporterError) => void
+  }[]
+  enc: OTLPExporterNodeConfigBasePlusEncryption['encryption'] | undefined
+  constructor (config: OTLPExporterNodeConfigBasePlusEncryption = {}) {
+    super(config)
+    this.enc = config.encryption
+    this.delayedItemsToExport = []
+    if (this.enc) {
```

Comment:
should this be turned on by default or is that handled in the config?

Reply:
do we only send requests if encryption is enabled? Otherwise we assume its the no op?

Reply:
encryption is enabled by default when telemetry is initialized by in `start-cypress.js`

if encryption is disabled we still send requests, this allows people to develop with telemetry locally.

in the binary encryption is always enabled.